test-utils/build.gradle

```diff
@@ -53,7 +53,7 @@ dependencies {
     implementation group: 'org.gradle', name: 'gradle-tooling-api', version: '7.3'
     implementation group: 'org.jetbrains', name: 'annotations', version: "17.0.0"
     implementation group: 'com.amazonaws', name: 'aws-java-sdk-s3', version: '1.12.425'
-    implementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-csv', version: '2.13.2'
+    implementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-csv', version: '2.13.5'
```

Comment:
We could try to upgrade to 2.14.1 instead, because we already get the transitively.

Reply:
That's not correct either actually. According to `./gradlew test-utils:dependencies` the one it gets transitively is `2.15.0`.

Reply:
Oh I might have checked with `./gradlew core:dependencies`, but that is even better